Tasks/PackerBuild/src/taskParameters.ts

```diff
@@ -51,21 +51,35 @@ export default class TaskParameters {
                     this._extractImageDetails();
                 }              
 
-                this.packagePath = this._getResolvedPath(tl.getVariable('System.DefaultWorkingDirectory'), tl.getInput(constants.DeployPackageInputName, true));
-                console.log(tl.loc("ResolvedDeployPackgePath", this.packagePath));
-                var deployScriptAbsolutePath = this._getResolvedPath(this.packagePath, tl.getInput(constants.DeployScriptPathInputName, true));
-                var scriptRelativePath = path.relative(this.packagePath, deployScriptAbsolutePath);
-                this.deployScriptPath = this._normalizeRelativePathForTargetOS(scriptRelativePath);
-                console.log(tl.loc("ResolvedDeployScriptPath", this.deployScriptPath));
+                try {
```

Comment:
Agree. I was hung-up about getting better error messages about these three blocks and hence added separate try-catch blocks. Will change it.